xbmc/epg/EpgDatabase.cpp

```diff
@@ -200,68 +219,81 @@ int CEpgDatabase::Get(CEpgContainer &container)
-        newTag->m_strDirector        = m_pDS->fv("sDirector").get_asString().c_str();
-        newTag->m_strWriter          = m_pDS->fv("sWriter").get_asString().c_str();
-        newTag->m_iYear              = m_pDS->fv("iYear").get_asInt();
-        newTag->m_strIMDBNumber      = m_pDS->fv("sIMDBNumber").get_asString().c_str();
-        newTag->m_iGenreType         = m_pDS->fv("iGenreType").get_asInt();
-        newTag->m_iGenreSubType      = m_pDS->fv("iGenreSubType").get_asInt();
-        newTag->m_genre              = StringUtils::Split(m_pDS->fv("sGenre").get_asString().c_str(), g_advancedSettings.m_videoItemSeparator);
-        newTag->m_iParentalRating    = m_pDS->fv("iParentalRating").get_asInt();
-        newTag->m_iStarRating        = m_pDS->fv("iStarRating").get_asInt();
-        newTag->m_bNotify            = m_pDS->fv("bNotify").get_asBool();
-        newTag->m_iEpisodeNumber     = m_pDS->fv("iEpisodeId").get_asInt();
-        newTag->m_iEpisodePart       = m_pDS->fv("iEpisodePart").get_asInt();
-        newTag->m_strEpisodeName     = m_pDS->fv("sEpisodeName").get_asString().c_str();
-        newTag->m_iSeriesNumber      = m_pDS->fv("iSeriesId").get_asInt();
-        newTag->m_strIconPath        = m_pDS->fv("sIconPath").get_asString().c_str();
+        newTag->m_iUniqueBroadcastID = m_pDS->fv(7).get_asInt();
+        newTag->m_iBroadcastId       = m_pDS->fv(8).get_asInt();
+        newTag->m_strTitle           = m_pDS->fv(9).get_asString().c_str();
+        newTag->m_strPlotOutline     = m_pDS->fv(10).get_asString().c_str();
+        newTag->m_strPlot            = m_pDS->fv(11).get_asString().c_str();
+        newTag->m_strOriginalTitle   = m_pDS->fv(12).get_asString().c_str();
+        newTag->m_strCast            = m_pDS->fv(13).get_asString().c_str();
+        newTag->m_strDirector        = m_pDS->fv(14).get_asString().c_str();
+        newTag->m_strWriter          = m_pDS->fv(15).get_asString().c_str();
+        newTag->m_iYear              = m_pDS->fv(16).get_asInt();
+        newTag->m_strIMDBNumber      = m_pDS->fv(17).get_asString().c_str();
+        newTag->m_iGenreType         = m_pDS->fv(18).get_asInt();
+        newTag->m_iGenreSubType      = m_pDS->fv(19).get_asInt();
+        newTag->m_genre              = StringUtils::Split(m_pDS->fv(20).get_asString().c_str(), g_advancedSettings.m_videoItemSeparator);
+        newTag->m_iParentalRating    = m_pDS->fv(21).get_asInt();
+        newTag->m_iStarRating        = m_pDS->fv(22).get_asInt();
+        newTag->m_bNotify            = m_pDS->fv(23).get_asBool();
+        newTag->m_iEpisodeNumber     = m_pDS->fv(24).get_asInt();
+        newTag->m_iEpisodePart       = m_pDS->fv(25).get_asInt();
+        newTag->m_strEpisodeName     = m_pDS->fv(26).get_asString().c_str();
+        newTag->m_iSeriesNumber      = m_pDS->fv(27).get_asInt();
+        newTag->m_strIconPath        = m_pDS->fv(28).get_asString().c_str();
 
```

Comment:
What's the advantage of using the column index rather than column name - performance? I'd have thought you should be using the column name whenever possible (and if a value in the result set doesn't have a column name you can give it one, ie. `0 as your_column_name`)